deploy/jiva-csi.yaml

```diff
@@ -281,11 +281,19 @@ spec:
             # This count has been set to 10 for sanity test cases as it takes
             # time in minikube
             - "--retrycount=10"
+            # metricsBindAddress is the TCP address that the controller should bind to
+            # for serving prometheus metrics. Change the port if 9505 is already in use.
+            # Remove the flag to disable prometheus metrics.
+            - "--metricsBindAddress=$(OPENEBS_POD_IP):9505"
```

Comment:
OPENEBS_POD_IP env is not required

Reply:
removing the OPENEBS_POD_IP means using `localhost` for metrics and using `localhost` is not a good practice as many test servers with different ports can already be present. cc @payes